### PR TITLE
Added hindi translation for the curl API rest guide

### LINKED ISSUE #66: Create a Hindi translation of the curl REST API guide
Based on the English version of the guide (https://github.com/devadvance/terminalcheatsheet/blob/staging/guides/curl-rest-api.md), create a Hindi translation in https://github.com/devadvance/terminalcheatsheet/blob/staging/i18n/hi/guides/curl-rest-api.md.

It is OK to use the English images/GIFs.

Please comment on this issue before starting work so people are informed that you are working on it.

### PR DESCRIPTION
**Pull request by IMPrimph**
## Description

_Please include a summary of this change and the issue(s) you are fixing. Include any context or information that will help a reviewer verify this PR quickly. List any caveats or issues with this change up front._

Fixes #66 

## Required checklist:

* [ ] My code matches the existing code style of this project.
* **One** of these:
	* [ ] This is a new localization for existing content.
	* [ ] This is new content that is localized for all the locales that this project currently supports.
	* [ ] I described above how I plan to localize this content immediately after merging these changes.
	* [ ] Localization is not applicable to this change.
* [ ] I have tested my code and it successfully builds for GitHub Pages and validates as AMP.

### CODE REVIEW

File: i18n/hi/guides/curl-rest-api.md [hunk 1]
```diff
@@ -7,9 +7,153 @@ permalink_without_prefix: /guides/curl-rest-api
...
+
+अपनी टर्मिनल विंडो में, `curl https: // jsonplaceholder.typicode.com / todos / 1` टाइप करें और **Enter ** दबाएं। आपको इसके समान परिणाम का अनुभव करना चाहिए:
+
+<div class="center guideimages">
+  <amp-anim src="/assets/guides/curl-rest-api/curl-get-basic-en.gif" width="665" height="387" alt="Demo of basic curl command" layout="responsive"></amp-anim>
+</div>
+
+किसी भी विकल्प के बिना, ** curl ** सर्वर के साथ बातचीत करने के लिए कर्ल डिफॉल्ट ** GET** HTTP अनुरोध विधि, जो आमतौर पर केवल डेटा पढ़ने के लिए उपयोग की जाती है। हम इस गाइड में बाद में अन्य HTTP अनुरोध विधियों को कवर करेंगे।
+
+चलो टर्मिनल विंडो में सीधे दिखाने के बजाय आउटपुट को फ़ाइल के रूप में सहेजने के लिए `-ओ` विकल्प जोड़ें। टाइप करें 'curl -o test.json https: // jsonplaceholder.typicode.com / todos / 1` और ** Enter** दबाएं:
```
Comment (devadvance) [inline i18n/hi/guides/curl-rest-api.md:53]: `-o` should not be translated here.

**Review by devadvance — CHANGES_REQUESTED** [review]
This translation is _nearly_ complete! A few areas to improve before merging:

* At the top of the file, the title and excerpt need a translation (replace the `TODO`).
* The headings (lines beginning with `##` should be translated where applicable.
* Make sure you're not translating the commands or options (like `-o`) since those don't need to be translated.

File: i18n/hi/guides/curl-rest-api.md (hunk 1 above)
Comment (IMPrimph) [reply i18n/hi/guides/curl-rest-api.md:53]: Extremely sorry, that was unexpected, I will change that :)

**Review by IMPrimph — COMMENTED** [review]
Made the changes specified

**Comment by IMPrimph** [comment]
Done!
@devadvance

**Review by devadvance — APPROVED** [review]
Looks good to me. Thanks for adding this valuable translation!

**Pull request merged**